lighthouse-core/test/results/sample_v2.json

```
@@ -7072,6 +7072,7 @@
       "runtimeSettingsUANetwork": "User agent (network)",
       "runtimeSettingsUrl": "URL",
       "runtimeUnknown": "Unknown",
+      "showRelevantAudits": "Show audits relevant to: ",
```

```suggestion
      "showRelevantAudits": "Show audits relevant to:",
```

I think? we can just do a space in html